test/index.js

```diff
@@ -28,12 +28,12 @@ const runTest = function(url) {
         error: '>=2000'
       },
       psi: {
-        warn: '>=1500',
-        error: '>=2000'
+        warn: '>=2000',
+        error: '>=3500'
       },
       vc85: {
-        warn: '>=1500',
-        error: '>=2000'
+        warn: '>=3500',
+        error: '>=5000'
```

Comment:
I'm hoping that's `ms` rather than `s` ?! ;)
Could it be related to the GH API simply being slow? (or does caching already hide any delays there?)